tac/project-updates/2024/2024-Q4-Hyperledger-Iroha.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: default
+title: 2024 Q4 Hyperledger Iroha
+parent: 2024
+grand_parent: Project Updates
+---
+
+| Iroha 2: [website](https://iroha.tech/) | Iroha 2: [documentation](https://docs.iroha.tech/)  |
+| :---- | :---- |
+
+# Project Health
+
+In the past quarter, Iroha 2 has made solid progress. The Iroha 2  team has released a pre-MVP version which is [RC 22.2](https://github.com/hyperledger-iroha/iroha/releases/tag/v2.0.0-pre-rc.22.2), and have been focusing on improving performance, refining the API, and updating SDKs and documentation. We're gearing up to release the MVP in the coming weeks, which will feature SDKs for [Java](https://github.com/hyperledger-iroha/iroha-java) and [Python](https://github.com/hyperledger-iroha/iroha-python); a [JavaScript/TS](https://github.com/hyperledger-iroha/iroha-javascript) SDK will follow in a later release.
+
+Additionally, the Iroha 2 team has finished developing the Iroha 2 [Blockchain Explorer](https://github.com/soramitsu/iroha2-block-explorer-web). The MVP is now in great shape and ready for use.
+
+We also successfully moved Iroha 2’s codebase to its [own repository](https://github.com/hyperledger-iroha) as part of the transition from the Hyperledger Foundation to the Linux Foundation’s Decentralized Trust initiative.
+
+The Iroha 2 community is active and growing, with lots of interest from potential users and contributors. Questions are being answered regularly, five issues were assigned to the maintainers among community members, and  the PRs for the ones have been merged since the last update. Project updates are being sent every two weeks to keep everyone connected and informed.
+
+# Questions/Issues for the TAC
+
+None at this time.
+
+# Releases
+
+[2.0.0-pre-rc.22.0](https://github.com/hyperledger-iroha/iroha/releases/tag/v2.0.0-pre-rc.22.0)  
+[2.0.0-pre-rc.22.1](https://github.com/hyperledger-iroha/iroha/releases/tag/v2.0.0-pre-rc.22.1)  
+[2.0.0-pre-rc.22.2](https://github.com/hyperledger-iroha/iroha/releases/tag/v2.0.0-pre-rc.22.2)
+
+Please note: that we are still in the process of finalizing and publishing the MVP PR.  
+Summary of Activities in the Last Quarter related to Community management
+
+# Overall Activity in the Past Quarter
+
+Iroha 2 has been actively engaging with the community through various channels, including Telegram and Discord, where we assist with technical questions, gather feedback, and provide technical guidance. 
+
+The team also hosts a weekly open [community call](https://meet.jit.si/Iroha_biweekly), where we share the latest updates and address questions from community members. Additionally, Iroha 2 sends out a regular newsletter to the [Hyperledger mailing list](https://lists.lfdecentralizedtrust.org/g/iroha), keeping everyone informed on the latest development updates and releases. All while ensuring that communication follows our [code of conduct](https://lf-decentralized-trust.github.io/governance/governing-documents/code-of-conduct.html).
```

Comment:
It would be great to get these calls added to the LF infrastructure so that recordings can be kept and listened to.